src/features/registry.js

```diff
@@ -11,6 +11,10 @@ systemJSPrototype.get = function (id) {
   }
 };
 
+systemJSPrototype.getLoad = function (id) {
+  return this[REGISTRY][id];
```

Comment:
It's quite important that we design this feature not to spill the internal `load` records, but rather to provide the minimal interface surface area to do this.

We should extend the registry API where possible to support the right primitives for hot reloading rather than opening up this interface.